ob1k-cache/src/main/java/com/outbrain/ob1k/cache/LocalAsyncCache.java

```diff
@@ -114,7 +118,8 @@ public LocalAsyncCache() {
       return fromValue(res);
     } else {
       if (failOnMissingEntries) {
-        return fromError(new RuntimeException("result is missing from loader response."));
+        final String prefix = cacheName == null ? "" : cacheName + ": ";
+        return fromError(new RuntimeException(prefix + "result for " + key + " is missing from loader response."));
```

Comment:
I try to give as much information as I can in the log message. If someone uses a key that does not implement toString, I leave it to him to decide whether to implement toString and gain a more informative message, or not.